### ISSUE TITLE
Form validation accessibility

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#249]
- [x] valid checkboxes (at least) are full green when validated: label should stay black.
- [x] error / state messages should use icon
- [x] and be associated with input's label via an `id` and `aria-describedby`, probably.
